### DISCUSSION TITLE
How to get Auditlog?

### DISCUSSION THREAD

**Discussion by Makin9** [open-metadata/OpenMetadata#17895] [Q&A] [answered]
Now, Im tring to use the OSS version of OpenMetaData.

In the terms of security, I wanna check audit log such as activity and login history.
But I couldnt find API to get them, so please help me by teaching the way to get them.

**Comment by Prajwal214** [comment, marked as answer]
@Makin9 You can fetch the logs under below path in the OpenMetadata Server
`/opt/openmetadata/logs`

<img width="219" alt="image" src="https://github.com/user-attachments/assets/b9f5ac4b-bd20-4d76-b090-1704ef998705">

**Reply by Makin9** [reply]
@Prajwal214 
Thank you very much!
Your reply was a big help to me, I was able to get the audit log and check it.

If you know, could you tell me whether the audit log has the history about users login?
I checked the audit log and I couldn't find any logs related to the login API.

**Reply by Prajwal214** [reply]
@Makin9 You can find login related information in the `openmetadata.log` log file

**Reply by Makin9** [reply]
@Prajwal214 
Thank you, I was able to check that the log has login related information.
Am I right to say, it is impossible to know who and when logged into OpenMetaData?
